docs/reference/ImageDraw.rst

```diff
@@ -285,6 +285,20 @@ Methods
 
         .. versionadded:: 5.3.0
 
+.. py:method:: ImageDraw.rounded_rectangle(xy, radius=0, fill=None, outline=None, width=1)
+
+    Draws a rounded rectangle.
+
+    :param xy: Two points to define the bounding box. Sequence of either
+            ``[(x0, y0), (x1, y1)]`` or ``[x0, y0, x1, y1]``. The second point
+            is just outside the drawn rectangle.
+    :param radius: Radius of the corners.
+    :param outline: Color to use for the outline.
+    :param fill: Color to use for the fill.
+    :param width: The line width, in pixels.
+
+        .. versionadded:: 8.2.0
```

Comment:
Looks like this directive is applied to the `width` param:

![image](https://user-images.githubusercontent.com/1324225/110236657-abfbb880-7f3f-11eb-8ba2-895540c87915.png)

Shall we move it up, similar to `ImageDraw.textlength`, to apply to the whole method?

![image](https://user-images.githubusercontent.com/1324225/110236696-eebd9080-7f3f-11eb-84ae-b740fcea9fe4.png)

Reply:
Thanks, done.

Reply:
Did this get missed in the rebase?

Reply:
I don't believe so. I suspect if you refresh https://pillow--5208.org.readthedocs.build/en/5208/reference/ImageDraw.html again, it will work.

Reply:
Ah yes, refresh sorted it, thanks!

![image](https://user-images.githubusercontent.com/1324225/110319113-17648980-8017-11eb-944c-6dc17cec1813.png)